src/windows/sign.c

```diff
@@ -267,7 +270,10 @@ static void window_sign_mouseup(rct_window *w, int widgetIndex)
 	}
 }
 
-/* rct2: 0x6B9784 & 0x6E6164 */
+/**
+ *
+ *  rct2: 0x6B9784
```

Comment:
Can you change this to the two addresses separated by a comma to match other functions that are based on multiple functions